tests/test_enhanced_slash_middleware_claude_dir.py

```
@@ -0,0 +1,67 @@
+from codex_plus.llm_execution_middleware import create_llm_execution_middleware
+
+
+def write_cmd(path: Path, name: str, description: str = "Test cmd") -> Path:
+    cmd_dir = path
+    cmd_dir.mkdir(parents=True, exist_ok=True)
+    cmd_file = cmd_dir / f"{name}.md"
+    cmd_file.write_text(f"---\ndescription: {description}\n---\n\nBody\n", encoding="utf-8")
+    return cmd_file
+
+
+def test_find_command_file_from_claude_directory(tmp_path):
```

Ruff (0.13.1)

Unused function argument: `tmp_path`

(ARG001)